libbeat/processors/add_cloud_metadata/docs/add_cloud_metadata.asciidoc

```diff
@@ -154,6 +159,9 @@ _Azure Virtual Machine_
 [source,json]
 -------------------------------------------------------------------------------
 {
+  "azure": {
+    "resourcegroup.name": "/subscriptions/641ebacb-7743-41e7-b4fa-af1167351a61/resourcegroups/testgroup/providers/Microsoft.ContainerService/managedClusters/testcluster"
```

Comment:
Field names should not contain dots when emitted from Beats. Restructure the event so that it produces `{azure: {resourcegroup: {name: ...`

update: I didn't see any incorrect code. So I think this is just a doc issue.